### PR TITLE
feat: add KeplrWalletConnectModal

### PR DESCRIPTION
**Pull request by flagrede**
## Description

UI for: regen-network/regen-web#2159

- add UI for web2 users trying to do transactions

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by flagrede** [comment]
@erikalogie you can check the UI part of this story here: https://deploy-preview-2173--regen-storybook.netlify.app/?path=/story/modal-keplrwalletconnectmodal--basic

**Comment by erikalogie** [comment]
> @erikalogie you can check the UI part of this story here: https://deploy-preview-2173--regen-storybook.netlify.app/?path=/story/modal-keplrwalletconnectmodal--basic

Looks good except for the link should be green, but not sure if that needs to be in storybook. The links should go here FYI: https://guides.regen.network/guides/wallets/wallet-setup/create-a-keplr-wallet

**Comment by erikalogie** [comment]
> @erikalogie Oh yes that was just an issue with the story, I fixed that. Also, the link will not be set directly in storybook.

great

**Comment by flagrede** [comment]
@blushi  Should we merge this UI PR before doing the rest?

File: web-components/.eslintrc.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ module.exports = {
     {
       files: ['*.tsx', '*.ts'],
       rules: {
+        'react-hooks/rules-of-hooks': 'off',
```
Comment (blushi) [inline web-components/.eslintrc.js:10]: should we just turn this off for the new lines in the stories that cause the issue?

File: web-components/src/components/modal/keplr-wallet-connect-modal/KeplrWalletConnectModal.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+import OutlinedButton from '../../../components/buttons/OutlinedButton';
+import { Body } from '../../../components/typography';
+import { KeplrWalletConnectModal } from './KeplrWalletConnectModal';
+
+export default {
+  title: 'Modal/KeplrWalletConnectModal',
+  component: KeplrWalletConnectModal,
+  argTypes: {
+    onClick: { action: 'clicked' },
```
[the remark this answers is not in the document]
Comment (flagrede) [reply web-components/src/components/modal/keplr-wallet-connect-modal/KeplrWalletConnectModal.stories.tsx:13]: I wanted this part to be customizable completely but I guess only sending the data using a `ButtonType` and keeping the UI here should be good enough. I updated that here: [51769a4](https://github.com/regen-network/regen-web/pull/2173/commits/51769a46cd913535ec251259f6e781678d942ed4)

**Review by blushi — APPROVED** [review]
looks good!

**Pull request merged**